### ISSUE TITLE
High CPU Usage in Aerospace When Zoom Workspace Is in the Background

### ISSUE THREAD

**Issue by Hisssy** [nikitabobko/AeroSpace#527]
When Zoom workspace is moved to the backgroud by Aerospace, the CPU usage of Aerospace increases from 1% to 44%.
Foreground:
<img width="1273" alt="图片" src="https://github.com/user-attachments/assets/f1d29ec9-cf43-4970-8273-f8152de21e97">
Background:
<img width="529" alt="图片" src="https://github.com/user-attachments/assets/bc56f38d-25e6-412a-bc0d-a9c6a4afd79e">

The Zoom workspace window seems to flicker in the corner of the screen. I suspect that Zoom has some mechanism to prevent it from being completely moved off-screen. However, Aerospace may not recognize this and repeatedly tries to move the window, causing the flickering and the high cpu usage.
Screen recording:

https://github.com/user-attachments/assets/89ce1ae1-5f2c-4860-ae0c-56603a575f62

## Checklist

- [x] I've searched for duplicates. My search queries were (please don't forget to search in closed issues too):
  - zoom
  - zoom flicker
- [x] I've read the guide https://nikitabobko.github.io/AeroSpace/guide
- [x] I've searched in the documentation (https://nikitabobko.github.io/AeroSpace/guide & https://nikitabobko.github.io/AeroSpace/commands). My search queries were:
  - dvorak
  - monitor
  - ...

```
aerospace CLI client version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
AeroSpace.app server version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
```

**Comment by xinkuifeng** [comment]
Hello @nikitabobko 

Is it possible to make a release with the fix so that we can upgrade Aerospace through Homebrew? 
Thanks

**Comment by nikitabobko** [comment]
Released in 0.15.0

**Comment by xinkuifeng** [comment]
On my side, confirmed that 0.15.2-Beta fixed this issue.
Thanks a lot!
